File: app/src/main/java/com/robotnec/budget/core/mvp/view/AddCategoryView.java
package com.robotnec.budget.core.mvp.view;

import com.robotnec.budget.core.domain.Category;

/**
 * @author zak <zak@swingpulse.com>
 */
public interface AddCategoryView extends View {
    void initEditMode(Category category);

    void closeView();
}

File: app/src/main/java/com/robotnec/budget/core/mvp/view/AddCategoryView.kt
package com.robotnec.budget.core.mvp.view

import com.robotnec.budget.core.domain.Category

/**
 * @author zak zak@swingpulse.com>
 */
interface AddCategoryView : View {
    fun initEditMode(category: Category)

    fun closeView()
}
